torch/_inductor/codegen/wrapper.py

```diff
@@ -1062,6 +1072,32 @@ def codegen_model_constructor(self):
                         f"inputs_info_[{idx}].shape.emplace_back({size}, {size}, nullptr);"
                     )
 
+            constant_offset_mapping = dict(V.graph.aot_constant_offset)  # noqa: C416
```

Comment:
Wondering why do we need ```# noqa: C416```?

Reply:
flake8 doesn't let me pass without this :(